dotnet/src/IntegrationTests/testsettings.json

```diff
@@ -30,5 +30,9 @@
   },
   "Postgres": {
     "ConnectionString": ""
+  },
+  "MongoDB": {
+    "ConnectionString": "",
+    "VectorSearchCollection": "dotnetMSKNearestTest.nearestSearch"
```

Comment:
@BorisDog  How do I set up this collection for testing?

Reply:
You would need to setup the following index via Atlas UI:
```
{
  "mappings": {
    "dynamic": true,
    "fields": {
      "embedding": {
        "dimensions": 3,
        "similarity": "cosine",
        "type": "knnVector"
      }
    }
  }
}
```
The collection is populated in `MongoDBMemoryStoreTestsFixture.InitializeAsync()`
Or alternatively you could use the Atlas cluster that was set up for these integration tests.